src/app/work-item/work-item-detail/work-item-detail.component.ts

```diff
@@ -1,15 +1,14 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { ActivatedRoute, Params } from '@angular/router';
-import { Location }               from '@angular/common';
 
```

Comment:
It appears that this file did not get rebased but instead reset to before the angular 2 update.